src/UniversalRewardsDistributor.sol

```diff
@@ -98,9 +97,9 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
         root = pendingRoot.root;
         ipfsHash = pendingRoot.ipfsHash;
 
-        emit EventsLib.RootSet(pendingRoot.root, pendingRoot.ipfsHash);
-
         delete pendingRoot;
```

Comment:
We should use `_setPendingRoot` here right?

Reply:
(outdated)